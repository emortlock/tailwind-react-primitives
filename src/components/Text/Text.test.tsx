import React from 'react'
import { render, RenderResult } from '@testing-library/react'

import { Text } from '.'

const setup = (
  testProps = {},
): [RenderResult, { props: Record<string, any> }] => {
  const props = Object.assign({}, testProps)

  const wrapper = render(<Text {...props}>Hello World</Text>)

  return [
    wrapper,
    {
      props,
    },
  ]
}

describe('Text', () => {
  it('renders matching snapshot', () => {
    const [{ asFragment }] = setup()

    expect(asFragment()).toMatchSnapshot()
  })
})
