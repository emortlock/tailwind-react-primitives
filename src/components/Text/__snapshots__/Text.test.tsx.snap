// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders matching snapshot 1`] = `
<DocumentFragment>
  <p
    class="leading-normal"
  >
    Hello World
  </p>
</DocumentFragment>
`;
